client/src/utils/algolia-locale-setup.ts

```diff
@@ -26,30 +26,30 @@ const algoliaIndices = {
     searchPage: 'https://www.freecodecamp.org/italian/news/search/'
   },
   portuguese: {
-    name: 'news-pt',
+    name: 'news-pt-br',
```

Comment:
Hi @sidemt, thanks for taking a look at this. Sorry, just seeing your comment now.

Yes, the index names should match what we have over on News. In this case the Portuguese index name on Algolia is `news-pt-br`, and is set correctly on News, but not on Learn.

There are some exceptions like German where we build Learn in German but not News, and in those cases it's best to show English search results because there are no articles to link to and no search page.